conf/buildpack.conf

```diff
@@ -33,5 +33,6 @@ tidy_version="5.6.0"
 sodium_php72_php73_version="1.0.7"
 sodium_version="1.0.18"
 coreutils_version="8.30"
+webp_version="1.2.0"
```

Comment:
@johnsudaar Where does this version number comes from? How can we know if an update is released?

Reply:
It comes from here: https://storage.googleapis.com/downloads.webmproject.org/releases/webp/index.html not sure how to be alerted :/

Reply:
Maybe we should add this link as a comment here? So that in case we want to update it, we know how to do it? 🤔